.tekton/pull-request.yaml

```diff
@@ -74,6 +74,25 @@ spec:
                 python -m pip install gitlint
                 git fetch --unshallow
                 gitlint --commits "origin/$(params.target-branch)..HEAD"
+      - name: yaml-lint
+        workspaces:
+          - name: workspace
+            workspace: workspace
+        runAfter:
+          - clone-repository
+        taskSpec:
+          workspaces:
+            - name: workspace
+          steps:
+            - name: yaml-lint
+              image: registry.access.redhat.com/ubi9/python-39:latest
+              workingDir: $(workspaces.workspace.path)
+              script: |
+                #!/bin/bash -ex
+
+                python -m pip install yamllint
+                cd $(workspaces.workspace.path)
```

Comment:
Do we need this line if we already specify `workingDir`?

Reply:
just checking here to see if that is the root path is correct

Reply:
The root path is correct and removed it.